### ISSUE TITLE
Update `UploadForm` component to accept `.zip`

### ISSUE THREAD

**Issue by kirkkwang** [notch8/viva#276]
# Story

Update the `UploadForm` component to accept zip files in addition to csv files.

# Acceptance Criteria

- [ ] the user can upload csv files still ([test.csv](https://github.com/user-attachments/files/16289661/viva_just_csv_test.csv))

- [ ] the user can now upload zip files as well ([test.zip](https://github.com/user-attachments/files/16289590/viva_test_cat_scan.zip))

# Screenshots / Video

![Image](https://github.com/user-attachments/assets/c3047d19-4c4a-4049-9a99-2b38046906a4)

# Testing Instructions and Sample Files

- [ ] go to `/uploads` route
- [ ] click on `No file chosen` (see acceptance criteria for files)
- [ ] observe that you can select a csv file
- [ ] observe that you can select a zip file

# Notes
Basically just turn this line into
```jsx
            accept='.csv,.zip'
```
https://github.com/scientist-softserv/viva/blob/main/app/javascript/components/ui/UploadForm/UploadForm.jsx#L19

Also adjust this:
https://github.com/scientist-softserv/viva/blob/main/app/javascript/components/pages/Uploads/index.jsx#L27
